uint16 arb_id
uint8 sub_id
uint8[] data